showpass/Showpass-E2E-tests
# SPW-9506 Ensure smooth transition between React and Angular and back.

**Description:**
For some reason, when a user logs into the portal, adds tickets to the cart and navigates to the checkout on AngularJS, they are required to authenticate again or continue as a guest. This is happening occasionally . During transition from React to Angular the session doesn't persist, which leads to a different workflow and the test fails.
Expected:
User should not be required to log in again when going through checkout.
Actual:
User session gets terminated and the 'Check out as a gest' button shows up.

```
Error 401 gets returned for 
api/auth/profile/
api/user/venues/employments/
```

Full video recording:
https://cloud.cypress.io/projects/e9axi7/runs/ee12673c-545d-466b-a616-ef3e327ae149/test-results/6d8773d1-5971-40e0-a430-945b756fb51c/replay?att=1

**Jira ticket:**
https://showpass.atlassian.net/browse/SPW-9506

**How to test:**
Run all the specs and ensure all tests pass, especially those that involve ticket purchasing.

**Test execution results:**
All tests passed:
https://cloud.cypress.io/projects/e9axi7/runs/93/overview?roarHideRunsWithDiffGroupsAndTags=1
